src/Microsoft.DotNet.Darc/Darc/Options/VirtualMonoRepo/VmrCommandLineOptionsBase.cs

```diff
@@ -25,11 +26,13 @@ protected IServiceCollection RegisterServices(string? tmpPath)
         var gitHubToken = GitHubPat;
         var azureDevOpsToken = AzureDevOpsPat;
 
+        // Read tokens from local settings if not provided
+        // We silence errors because the VMR synchronization often works with public repositories where tokens are not required
         if (gitHubToken == null || azureDevOpsToken == null)
         {
             try
             {
-                localDarcSettings = LocalSettings.LoadSettingsFile(this);
+                localDarcSettings = LocalSettings.GetSettings(this, NullLogger.Instance);
```

Comment:
what if there are actual errors that we'd want to see, like missing perms? will we also silence those?

Reply:
I think overall this is not very likely. You'd have your darc broken already and would have to call the VMR commands which no one does locally. But this code will go away at the end of the month hopefully.